package.json

```diff
@@ -9,9 +9,10 @@
     "check:release": "scripts/check-packages.sh",
     "check:component": "run-s clean:php \"render:* {1} {2}\" pretty:php \"diff {1} {2}\" --",
     "copy:static": "ncp static build",
-    "copy:php": "node_modules/@ecl-twig/php-storybook/resources/get-assets.js",
+    "copy:php": "./node_modules/@ecl-twig/php-storybook/resources/get-assets.js",
     "clean": "rimraf build dist",
     "clean:php": "rimraf build php/packages php/dist",
+    "deploy:php": "gh-pages -d php/dist",
```

Comment:
Ok, removed the duplicated line, re-added the dep, this deployment was set on github pages just by convenience, it was really straightforward but yes, it makes much more sense to have it on netlify, to be honest i haven't done that because i don't event have access to the current "ecl-twig" site configuration on netlify.
Let's do it in another ticket, though, i'll create it ;)

Reply:
Created https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1236
Feel free to use your personal account for the new sites. There is no reason why not to have full management rights on these projects.